docs/troubleshooting.md

```diff
@@ -0,0 +1,127 @@
+
+If everything seems healthy, and still no packages show up in the UI, it could be a browser issue. Check the the browser console to see the logs for possible errors. At any point in the steps above if the error looks too complicated to debug, please reach out to the #forum-operators channel in the [upstream Kubernetes slack instance](https://slack.k8s.io/) or the [google group for Operator-Framework](https://groups.google.com/forum/#!forum/operator-framework). 
+
+
+
+## OperatorSource failing to download from datastore
+
+If an OperatorSource fails to reconcile for any reason (i.e operators in that particular OperatorSource fail to show up in the UI), it might be helpful first inspect the `STATUS` and the `MESSAGE` of the OperatorSource with `oc get opsrc -n openshift-marketplace`. You can also inspect the log messages of the marketplace-operator for detailed information about the OperatorSource. 
+
+Note: A common reason for an OperatorSource to not reconcile correctly has been the use of the wrong [end point](https://github.com/operator-framework/operator-marketplace/blob/master/deploy/examples/community.operatorsource.cr.yaml#L8). Please make sure that the end point is specified as `https://quay.io/cnr`.
+
+## OperatorSource `MESSAGE` reads unknown error (status 404)
+
+Some of the common reasons we have encountered that caused this `MESSAGE` are quay.io being unavailable or the wrong end point being listed in the OperatorSource. If the steps listed [here](#operatorSource-failing-to-download-from-datastore) does not help debug the problem, please reach out to the #forum-operators channel in the [upstream Kubernetes slack instance](https://slack.k8s.io/) or the [google group for Operator-Framework](https://groups.google.com/forum/#!forum/operator-framework). 
+
+## Cannot uninstall operator from the CLI in a clean way. 
+
+To uninstall an operator from the CLI, you need to first delete the Subscription for the operator, then delete the ClusterServiceVersion of the operator, and finally delete the CatalogSourceConfig for the operator. An example with detailed steps to delete an operator from the CLI is listed [here](https://github.com/operator-framework/operator-marketplace#uninstalling-an-operator-via-the-cli).  
+
+## CatalogSourceConfig stuck in Configuring phase
+
+If you are trying to [install an operator from the cli](https://github.com/operator-framework/operator-marketplace#installing-an-operator-using-marketplace), the packages listed in the CatalogSourceConfig must have valid package names, and the apps in the quay namespace must be made public. If you have an app resgistry repository backed by authentication, please follow the steps listed [here](https://github.com/operator-framework/operator-marketplace/blob/master/docs/how-to-authenticate-private-repositories.md).
```

Comment:
It should be `app-registries in the Quay namespace`. Moreover they needn't be public. 

Please say `private app-registries` instead of `backed by auth`